about-pses/example-pse.md

```diff
@@ -179,21 +206,25 @@ Write the logical steps here.
 ##### !end-placeholder
 
 ##### !explanation
+It may feel cumbersome to write out the steps in detail, but it can be a helpful exercise to think through the problem before writing code. This is also a great way to practice speaking _about_ coding, which we'll need to become comfortable with for interviews and other professional settings.
```

Comment:
Is it worth mentioning that in an interview setting, doing this step gives the interviewer insight into how you understand the problem, and provides an opportunity for them to give you more information before you are deep in an implementation if any part of the problem statement was misunderstood?